dask/dataframe/utils.py

```diff
@@ -657,7 +657,7 @@ def eq_dtypes(a, b):
         ) or (a == b)
 
     if not is_dask_collection(res) and is_dataframe_like(res):
-        for col, a, b in pd.concat([ddf._meta.dtypes, res.dtypes], axis=1).itertuples():
+        for _, a, b in pd.concat([ddf._meta.dtypes, res.dtypes], axis=1).itertuples():
```

Comment:
It looks like `itertuples` has an `index=` keyword we can set to `False` to exclude the unused variable (note: I've not tested the suggestion below)

```suggestion
        for a, b in pd.concat([ddf._meta.dtypes, res.dtypes], axis=1).itertuples(index=False):
```